src/Appwrite/Network/Validator/URL.php

```diff
@@ -49,11 +49,17 @@ public function getDescription(): string
      */
     public function isValid($value): bool
     {
-        if (\filter_var($value, FILTER_VALIDATE_URL) === false) {
+        $sanitized_url = '';
```

Comment:
Would you please update this to be camel cased to match our convention?

```suggestion
        $sanitizedURL = '';
```

Reply:
sure